### ISSUE TITLE
Styleguide component | Table

### ISSUE THREAD

**Issue by sneha-pai** [department-of-veterans-affairs/caseflow#8920]
### Context
Table component has been established in generic Queues, and styleguide table component has been outdated for quite some time.

### Related issues

#12094 –  Queue table headers not being read by screen reader
#12126 – Table sort functionality not accessible to keyboard-only or other AT users. Sort buttons should be included in the tab order and be operable by keyboard
